ibandit.gemspec

```diff
@@ -1,26 +1,28 @@
-require File.expand_path('../lib/ibandit/version', __FILE__)
+# frozen_string_literal: true
+
+require File.expand_path("../lib/ibandit/version", __FILE__)
 
 Gem::Specification.new do |gem|
-  gem.add_development_dependency 'rspec',           '~> 3.3'
-  gem.add_development_dependency 'rspec-its',       '~> 1.2'
-  gem.add_development_dependency 'rubocop',         '~> 0.33.0'
-  gem.add_development_dependency 'sax-machine',     '~> 1.3'
-  gem.add_development_dependency 'nokogiri',        '~> 1.6'
+  gem.add_development_dependency "nokogiri",        "~> 1.6"
+  gem.add_development_dependency "rspec",           "~> 3.3"
+  gem.add_development_dependency "rspec-its",       "~> 1.2"
+  gem.add_development_dependency "rubocop",         "~> 0.52.0"
+  gem.add_development_dependency "sax-machine",     "~> 1.3"
 
-  gem.add_runtime_dependency 'i18n', '~> 0.7.0'
+  gem.add_runtime_dependency "i18n", "~> 0.7.0"
 
-  gem.authors = ['Grey Baker']
-  gem.description = 'Ibandit is a Ruby library for manipulating and ' \
-                    'validating IBANs. It allows you to construct an IBAN ' \
-                    'from national banking details; deconstruct an IBAN into ' \
-                    'national banking details; and validate an IBAN\'s check ' \
-                    'digits and format.'
-  gem.email = ['grey@gocardless.com']
+  gem.authors = ["Grey Baker"]
+  gem.description = "Ibandit is a Ruby library for manipulating and " \
+                    "validating IBANs. It allows you to construct an IBAN " \
+                    "from national banking details; deconstruct an IBAN into " \
+                    "national banking details; and validate an IBAN's check " \
+                    "digits and format."
+  gem.email = ["grey@gocardless.com"]
```

Comment:
We should update this to `developers@gocardless.com`, like https://github.com/gocardless/coach/blob/master/coach.gemspec#L11.

Sorry @greysteil!

Reply:
Makes sense to me - I don't even have push access to this anymore!